content/docs/2.7/scalers/rabbitmq-queue.md

```diff
@@ -32,6 +33,7 @@ triggers:
 - `queueName` - Name of the queue to read message from.
 - `mode` - QueueLength to trigger on number of messages in the queue. MessageRate to trigger on the publish rate into the queue. (Values: `QueueLength`, `MessageRate`)
 - `value` - Message backlog or Publish/sec. rate to trigger on.
+- `activationValue` - Min value to activate the scaler. (Default: `0`, Optional)
```

Comment:
```suggestion
- `activationValue` - Target value for activating the scaler. Learn more about activation [here](./../concepts/scaling-deployments.md#activating-and-scaling-thresholds).(Default: `0`, Optional, This value can be a float)
```

Reply:
For the documentation, we are using the same definition for all the scalers, I have suggested it